client/src/RatingsAndReviews/Reviews/Reviews.jsx

```diff
@@ -58,19 +61,34 @@ class Reviews extends React.Component {
     });
   }
 
+  openAddReview() {
+    this.setState({
+      addReviewOpen: !this.state.addReviewOpen
+    });
+  }
+
   loadTwoMore() {
     this.setState({loaded: this.state.loaded + 2})
```

Comment:
I've read before that using setState to increment a state value can have errors in correctly incrementing because setting state can be updated asynchronously. If you aren't running into any issue, then I probably wouldn't worry about it-- or if you have already considered this and I'm just not seeing it then ignore. This article talks about what I'm thinking of-- https://reactjs.org/docs/state-and-lifecycle.html#state-updates-may-be-asynchronous